gpt4all-chat/chatllm.h

```diff
@@ -11,6 +11,7 @@ enum LLModelType {
     MPT_,
     GPTJ_,
     LLAMA_,
+    REPLIT_,
```

Comment:
This is an enum. Don't change the value of `CHATGPT_` like this by pushing it down just to order the lines by length. The base assumption should be that an enumeration has its current exact value for some reason, and even when it's arbitrary it's still safer to pretend as if it matters. Sort of like with guns: it's always loaded, even when it's not loaded.

Reply:
ah good call out, fixed. 
(I don't think the gun comment was necessary though)